server/models.ts

```diff
@@ -92,16 +92,8 @@ export const attributesPublicUser = [
 ];
 
 export const includeUserModel = (() => {
-	return ({
-		attributes: providedAttrs,
-		...restOptions
-	}: {
-		attributes?: string[];
-		[k: string]: any;
-	}) => {
-		const attributes = providedAttrs
-			? [...new Set([...attributesPublicUser, ...providedAttrs])]
-			: attributesPublicUser;
+	return ({ attributes: providedAttributes = [], ...restOptions }) => {
```

Comment:
This is stupid (and deserves more attention) but we cannot actually use TypeScript features in this file because it is required from the `stubstub` global setup without being touched by TS.